### ISSUE TITLE
process labels not on the same line are not detected as a process label

### ISSUE THREAD

**Issue by jeremiah-c-leary** [jeremiah-c-leary/vhdl-style-guide#257]
In this example:

```vhdl
   some_process_label:

   process () is

```

the some_process_label is not detected as a process label.  Maybe this should be a new rule when the process label should be on the same line at the process keyword.
